docs/api-reference/layer.md

```diff
@@ -288,6 +288,21 @@ And is expected to return an array of "ranges", in the form of `{startRow, endRo
 
 **This feature is experimental and intended for advanced use cases.** Note that it only rewrites part of a buffer, not remove or insert, therefore the user of this prop is responsible of making sure that all the unchanged objects remain at the same indices between `oldData` and `newData`. This becomes trickier when dealing with data of dynamic lengths, for example `PathLayer`, `PolygonLayer` and `GeoJsonLayer`. Generally speaking, it is not recommended to use this feature when the count of vertices in the paths/polygons may change.
 
+##### `positionFormat` (String, optional)
+
+One of `'XYZ'`, `'XY'`.
+
+This prop is currently only effective in `PathLayer`, `SolidPolygonLayer` and `PolygonLayer`.
+
+Default `'XYZ'`.
+
+##### `colorFormat` (String, optional)
+
+One of `'RGBA'`, `'RGB'`.
+
+Setting it to `'RGB'` will make the layer ignore the alpha channel and assume `a: 255`.
+
+Default `'RGBA'`.
```

Comment:
3D tiles also have [rgb565](https://github.com/uber-web/loaders.gl/blob/master/modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js#L215) color format. Do you think it could also be provided in deck.gl?

Reply:
No, that cannot be done without changing attribute generation.